### PR TITLE
feat: about page HP-2259

### PR DESCRIPTION
**Pull request by timwessman**
About page in three languages. 

[HP-2259](https://helsinkisolutionoffice.atlassian.net/browse/HP-2259)

[HP-2259]: https://helsinkisolutionoffice.atlassian.net/browse/HP-2259?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-UI branch is deployed to platta: https://helsinki-profile-pr325-ui.dev.hel.ninja** :rocket::rocket::rocket:

File: src/common/footer/Footer.tsx [hunk 1]
```diff
@@ -33,6 +33,7 @@ const Footer = () => {
       title={t('appName')}
     >
       <HDSFooter.Utilities>
+        <HDSFooter.Link as={Link} to="/about" label={t('footer.about')} />
```
Comment (mikkojamG) [inline src/common/footer/Footer.tsx]: According to ticket description, this should be located next to Accessibility document and Cookie settings. 🤔 Moving this inside `<HDSFooter.Base />` should do the trick

File: src/aboutPage/AboutPage.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import AboutPageFi from './AboutPageFi';
+import AboutPageSv from './AboutPageSv';
+import AboutPageEn from './AboutPageEn';
+import commonContentStyles from '../common/cssHelpers/content.module.css';
+import PageLayout from '../common/pageLayout/PageLayout';
+
+function AboutPage(): React.ReactElement {
+  const { i18n } = useTranslation();
+  const selectStatement = () => {
+    const lang =
```
Comment (mikkojamG) [inline src/aboutPage/AboutPage.tsx]: Could also use `i18n.language` 🤔

File: src/aboutPage/AboutPageEn.tsx [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+        If the use of the service requires official personal data, the customer
+        is offered strong authentication to the profile via Suomi.fi&apos;s
+        central authentication service so that official personal data can be
+        transmitted to the service.
+      </p>
+      <p>
+        As official personal data, the Helsinki profile stores the basic
+        information about the Customer obtained through Suomi.fi from the
+        Population Information System, which includes
+        <ul>
```
Comment (mikkojamG) [inline src/aboutPage/AboutPageEn.tsx]: I noticed that there is a console warning: `<ul> cannot appear as a descendant of <p>`. Should be an easy fix 🤔

**Review by mikkojamG — CHANGES_REQUESTED** [review]

File: src/aboutPage/AboutPage.tsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+import classNames from 'classnames';
+
+import styles from './AboutPage.module.css';
+import AboutPageFi from './AboutPageFi';
+import AboutPageSv from './AboutPageSv';
+import AboutPageEn from './AboutPageEn';
+import commonContentStyles from '../common/cssHelpers/content.module.css';
+import PageLayout from '../common/pageLayout/PageLayout';
+
+function AboutPage(): React.ReactElement {
```
Comment (mikkojamG) [inline src/aboutPage/AboutPage.tsx:12]: Looks like Sonar is complaining about test coverage on these About pages. There isn't much to test, could add simple tests just to see that everything renders without errors. 🙂

**Review by mikkojamG — CHANGES_REQUESTED** [review]

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-UI branch is deployed to platta: https://helsinki-profile-pr325-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-UI branch is deployed to platta: https://helsinki-profile-pr325-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-UI branch is deployed to platta: https://helsinki-profile-pr325-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-UI branch is deployed to platta: https://helsinki-profile-pr325-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-UI branch is deployed to platta: https://helsinki-profile-pr325-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Review by mikkojamG — APPROVED** [review]

**Comment by terovirtanen** [comment]
**HELSINKI-PROFILE-UI branch is deployed to platta: https://helsinki-profile-pr325-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Pull request merged**